database/schema/mysql-schema.sql

```diff
@@ -0,0 +1,1588 @@
+  `deleted_at` timestamp NULL DEFAULT NULL,
+  PRIMARY KEY (`ID`),
+  KEY `achievements_gameid_index` (`GameID`),
+  KEY `achievements_points_index` (`Points`),
+  KEY `achievements_trueratio_index` (`TrueRatio`),
+  KEY `achievements_user_id_foreign` (`user_id`),
+  KEY `achievements_type_index` (`type`),
+  KEY `achievements_game_id_published_index` (`GameID`,`Flags`),
+  CONSTRAINT `achievements_game_id_foreign` FOREIGN KEY (`GameID`) REFERENCES `GameData` (`ID`) ON DELETE CASCADE,
+  CONSTRAINT `achievements_user_id_foreign` FOREIGN KEY (`user_id`) REFERENCES `UserAccounts` (`ID`) ON DELETE SET NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_uca1400_ai_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+DROP TABLE IF EXISTS `Comment`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `Comment` (
+  `ID` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
+  `ArticleType` tinyint(3) unsigned NOT NULL,
+  `ArticleID` int(10) unsigned NOT NULL,
+  `user_id` bigint(20) unsigned DEFAULT NULL,
+  `commentable_type` varchar(255) DEFAULT NULL,
+  `commentable_id` bigint(20) unsigned DEFAULT NULL,
+  `Payload` text NOT NULL,
+  `Submitted` timestamp NOT NULL DEFAULT current_timestamp(),
+  `Edited` timestamp NULL DEFAULT NULL,
+  `deleted_at` timestamp NULL DEFAULT NULL,
+  PRIMARY KEY (`ID`),
+  KEY `comment_articleid_index` (`ArticleID`),
+  KEY `comments_commentable_index` (`commentable_type`,`commentable_id`),
+  KEY `comment_user_id_foreign` (`user_id`),
+  CONSTRAINT `comment_user_id_foreign` FOREIGN KEY (`user_id`) REFERENCES `UserAccounts` (`ID`) ON DELETE SET NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_uca1400_ai_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+DROP TABLE IF EXISTS `Console`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `Console` (
+  `ID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `Name` varchar(255) NOT NULL,
+  `name_full` varchar(255) DEFAULT NULL,
+  `name_short` varchar(255) DEFAULT NULL,
+  `manufacturer` varchar(255) DEFAULT NULL,
+  `active` tinyint(1) DEFAULT NULL,
+  `order_column` int(10) unsigned DEFAULT NULL,
+  `Created` timestamp NULL DEFAULT current_timestamp(),
+  `Updated` timestamp NULL DEFAULT current_timestamp() ON UPDATE current_timestamp(),
+  `deleted_at` timestamp NULL DEFAULT NULL,
+  PRIMARY KEY (`ID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_uca1400_ai_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+DROP TABLE IF EXISTS `EmailConfirmations`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `EmailConfirmations` (
+  `User` varchar(20) NOT NULL,
+  `user_id` bigint(20) unsigned DEFAULT NULL,
+  `EmailCookie` varchar(20) NOT NULL,
+  `Expires` date NOT NULL,
+  PRIMARY KEY (`id`),
```

Comment:
You [explicitly removed](https://github.com/RetroAchievements/RAWeb/pull/2598/commits/fb630ee5ffa7d4aff07722d9968514594f09e7e1#diff-9f611797ecac538e43ae8149ee92ced6b58c8587375deb6f505d395312288643L90) the `id` column from this table, but there's still a key reference.

Which I assume is responsible for this error doing a `migrate:fresh`:
```
Error Output:
================
mysql: [Warning] Using a password on the command line interface can be insecure.
ERROR 1072 (42000) at line 89: Key column 'id' doesn't exist in table
```